### PR TITLE
Made corrections according to comments

### PR DESCRIPTION
**Pull request by Ihnatenko-A**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +50,7 @@
     ]
   },
   "devDependencies": {
-    "@types/react-router-dom": "^5.1.7"
+    "@types/react-router-dom": "^5.1.7",
```
Comment (lazio) [inline package.json]: move to dependencies

File: src/components/SupplierPage/Component.tsx [hunk 2]
```diff
@@ -72,47 +30,68 @@ const initialSupplierState = {
   "revenue": '',
   "type": '',
   "website": '',
-}
+};
 
-const SupplierPage:FunctionComponent<any> = ({ match }: RouteComponentProps<TParams>) => {
+enum Tabs {
```
Comment (lazio) [inline src/components/SupplierPage/Component.tsx:35]: nice 👍

File: src/components/SupplierPage/components/MapTab/styles.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    padding: '20px 0'
+  },
+  heading: {
+    fontWeight: 'bold',
+    marginBottom: 12,
+  },
+  locationBlock: {
+    marginBottom: 40,
+  },
+  InfoBlock: {
```
Comment (lazio) [inline src/components/SupplierPage/components/MapTab/styles.ts]: remove

File: src/components/SuppliersList/Component.tsx [hunk 4]
```diff
@@ -8,91 +8,42 @@ import Paper from '@material-ui/core/Paper';
...
-  const [data, setData] = useState<IData>({});
-
   const [suppliers, setSuppliers] = useState<ISupplier[]>([]);
 
   useEffect(() => {
-    getSuppliers().then((res: any) => {
-      setData(res)
-    });
-  }, [])
+    const suppliersIds = Object.keys(data);
```
Comment (lazio) [inline src/components/SuppliersList/Component.tsx]: data - is bad name

File: package.json (hunk 1 above)
Comment (Ihnatenko-A) [reply package.json]: done

**Comment by lazio** [comment]
image
please remove header on a homepage

File: src/components/SupplierPage/components/ContactsTab/utils.ts [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    name: 'Sven Sven',
+    location: 'Curnovec, SI',
+    email: 'sven@brembo.com',
+    phone: '+1 12314123',
+    additional: 'Here',
+    id: '12',
+  },
+]
+
+export const useStyles = makeStyles((theme) => ({
```
Comment (lazio) [inline src/components/SupplierPage/components/ContactsTab/utils.ts]: move styles to styles.js file

Comment (Ihnatenko-A) [reply src/components/SupplierPage/components/ContactsTab/utils.ts]: done

File: src/components/SuppliersList/Component.tsx (hunk 4 above)
Comment (Ihnatenko-A) [reply src/components/SuppliersList/Component.tsx]: changed to suppliersData

File: src/components/SupplierPage/components/MapTab/styles.ts (hunk 3 above)
Comment (Ihnatenko-A) [reply src/components/SupplierPage/components/MapTab/styles.ts]: removed empty rule

**Pull request merged**